src/libcrun/linux.c

```diff
@@ -1631,8 +1632,19 @@ do_finalize_notify_socket (libcrun_container_t *container, libcrun_error_t *err)
   if (UNLIKELY (ret < 0))
     return ret;
 
+#ifdef HAVE_FGETXATTR
+  if (container->container_def->linux->mount_label)
+    {
+      cleanup_free char *socket_path_file = NULL;
+      xasprintf (&socket_path_file, "%s/notify", host_notify_socket_path);
+      /* Ignore the error, the worse that can happen is that the container fails to notify it is ready.  */
+      (void) setxattr (socket_path_file, "security.selinux", container->container_def->linux->mount_label,
```

Comment:
setfscreatecon(container->container_def->linux->mount_label) Before creating the socket.  Should use setfilecon() rather then calling setxattr directly.